.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Takes the version of apm-agent-java in the Dockerfile and creates a tag
+# if the tag does not exist yet.
+name: create-tag
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - Dockerfile
+
+permissions:
+  contents: write
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # also fetch tags
+      - run: |
+          APM_AGENT_JAVA_VERSION="v$(cat Dockerfile | grep "apm-agent-java" | awk '/apm-agent-java:/{ print $2 }' | awk -F: '{ print $2 }')"
```

Comment:
A Github workflow is a YAML file you can update it from updatecli and you can get rid of this logic, for example

```
 - run: |
     ....
    env:
      APM_AGENT_JAVA_VERSION: 1.1.1
```

In this case, you only have to update the env var `APM_AGENT_JAVA_VERSION` in the YAML file from update cli.

Another option is to keep a plaintext file in the root folder to .ci folder with the version of the Agent and get it from whatever process need it with a simple `cat`

Reply:
https://github.com/elastic/opbeans-java/pull/190/commits/148ce4d64d2601832d3781f2366009b5fc8c67f2